boudicca.base/eventdb/build.gradle.kts

```diff
@@ -7,5 +7,6 @@ dependencies {
     implementation("org.springframework.boot:spring-boot-starter-security")
     implementation(project(":boudicca.base:common-model"))
     implementation(project(":boudicca.base:eventdb-api"))
+    implementation(libs.logback)
```

Comment:
that would probably be a good dependency to put into our own gradle plugins?

Reply:
true, will move it